brushfire-core/src/main/scala/com/stripe/brushfire/TreeTraversal.scala

```diff
@@ -15,12 +16,14 @@ import com.twitter.algebird._
  * features). A tree traversal chooses which paths to go down (which may be all
  * of them) and the order in which they are traversed.
  */
-trait TreeTraversal[K, V, T, A] {
+trait TreeTraversal[Tree, K, V, T, A] {
```

Comment:
Since the alias was never actually removed I think things are probably fine as they are.